modules/rosa-create-dedicated-cluster-admins.adoc

```diff
@@ -15,15 +15,14 @@ As the user that created the cluster, add the `cluster-admin` user role to your
 
 .Procedure
 
-. To grant cluster access to a `dedicated-admin` user:
-.. Enter the following command to promote your user to a `dedicated-admin`:
+. Enter the following command to promote your user to a `dedicated-admin`:
 +
 [source,terminal]
 ----
 $ rosa grant user dedicated-admin --user <idp_user_name> --cluster <cluster_name>
 ----
 +
-.. Enter the following command to verify that your user now has `dedicated-admin` access. As a `dedicated-admin` you should see output similar to the example output when entering the following command:
+. Enter the following command to verify that your user now has `dedicated-admin` access. As a `dedicated-admin` you should see output similar to the example output when entering the following command:
```

Comment:
`dedicated admin` -> dedicated administrator
Per ISG, avoid "should"...and I don't think you need the second sentence at all.

Reply:
Done.